test/configCases/css/minimize-dead-rules/style.css

```
@@ -0,0 +1,52 @@
+.between5 { color: teal }
+.t05 { color: blue }
+
+/*! three copies leave the last */
+.t06 { color: red }
+.m1 { color: teal }
+.t06 { color: red }
+.m2 { color: teal }
+.t06 { color: red }
+
+/*! a block a cut empties goes, as an empty one written here would */
+@media speech { .t07 { color: red } }
```

Stylelint (17.14.0)

[error] Deprecated media type "speech" (media-type-no-deprecated)

(media-type-no-deprecated)